pkg/kubelet/cm/devicemanager/topology_hints_test.go

```diff
@@ -250,6 +271,65 @@ func TestGetTopologyHints(t *testing.T) {
 				},
 			},
 		},
+		{
+			description:   "Single device type, all already allocated to container",
```

Comment:
Just curious, can we hit this case in practice?

Reply:
For the exact case you've run into -- after a kubelet restart.

Reply:
Ah. That makes more sense.

Yeah, that case should never come up in practice today, but it's good to test to make sure that the logic does what's expected should any code change that affects this.